venues/AKBC.ws/2019/Conference/python/assign.py

```diff
@@ -1,7 +1,7 @@
 
 
 import openreview
-import akbc19
+import akbc19 as conferenceConfig
```

Comment:
really minor thing (which you don't have to follow if you don't want): the conventional style for python variables is to use snake_case instead of camelCase

Reply:
Thanks for this suggestion. I have made changes in accordance.